src/main/java/console/common/CRUDParseUtils.java

```diff
@@ -60,7 +60,7 @@ public static void parseCreateTable(String sql, Table table)
             keyFlag = true;
             Index index = indexes.get(0);
             String type = index.getType().toLowerCase();
-            if (type.equals("primary key")) {
+            if ("primary key".equals(type)) {
```

Comment:
will be better to define a constant for this.

Reply:
added it.